docs/docker/intro.md

```diff
@@ -0,0 +1,33 @@
+# Docker
+
+Docker is a container based virtualization software that allow it's user to create a development environment in a easy way without installing a virtualization software i.e VirtualBox.
```

Comment:
Docker is a container-based virtualization software that allows users to easily create a development environment without installing any virtualization software (e.g. VirtualBox).

Reply:
Done